examples/gke-security-best-practices/firewall.tf

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+/*
+  This file manages firewall resources
+*/
+
+//bastion firewall rule: all SSH from provided source network
```

Comment:
Should we add a rule to explicitly deny all other addresses?

Reply:
I don't believe so:
the firewall rule here explicit:

Direction Ingress
Action on match Allow
Targets Service account
bastion-dev-gke@scalesec-dev.iam.gserviceaccount.com
Source filters
IP ranges
97.106.112.101/32
Protocols and ports
tcp:22